docs/src/pages/components/autocomplete/autocomplete.md

```diff
@@ -167,9 +167,9 @@ A customized UI for Google Maps Places Autocomplete.
 
 {{"demo": "pages/components/autocomplete/GoogleMaps.js"}}
 
-For this demo, we need to load the [Google Maps JavaScript](https://developers.google.com/maps/documentation/javascript/tutorial) API.
+For this demo, we need to load the [Google Maps JavaScript](https://developers.google.com/maps/documentation/javascript/tutorial) and [Google Places](https://developers.google.com/maps/documentation/places/web-service/overview) API.
 
-> ⚠️ Before you can start using the Google Maps JavaScript API, you must sign up and create a billing account.
+> ⚠️ Before you can start using the Google Maps JavaScript API or Places API, you must sign up and create a billing account.
```

Comment:
```suggestion
> ⚠️ Before you can start using the Google Maps JavaScript API and Places API, you must sign up and create a billing account.
```

Isn't it _and_ here as well? Given that it seems that both APIs need to be used together.

Reply:
Ah thanks @danilo-leal 👍